sdk/python/pvsite_datamodel/sqlmodels.py

```diff
@@ -128,60 +138,33 @@ class ForecastValueSQL(Base, CreatedMixin):
 
     __tablename__ = "forecast_values"
 
-    forecast_value_uuid = sa.Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
-    datetime_interval_uuid = sa.Column(
-        UUID(as_uuid=True),
-        sa.ForeignKey("datetime_intervals.datetime_interval_uuid"),
-        nullable=False,
-        default=uuid.uuid4,
+    forecast_value_uuid = sa.Column(
+        UUID(as_uuid=True), default=uuid.uuid4, primary_key=True
     )
-    forecast_generation_kw = sa.Column(REAL, nullable=False)
+
+    start_utc = sa.Column(sa.DateTime, nullable=False, index=True)
+    end_utc = sa.Column(sa.DateTime, nullable=False)
+    power_kw = sa.Column(sa.Float, nullable=False)
```

Comment:
I found it very confusing that the predicted power and the historical power had different names, but I'm happy to revert the change if there was a reason for the different name.

Reply:
I thought it was good having slightly different names, so people dont get confused between a forecast and actual

Reply:
Yeah but since the table name is never going to be far, perhaps it's not that confusing? But what was even more confusing is that it's `Forecasts.forecast_generation_kw` but not `Generation.generation_kw`; at least it should be `Forecasts.forecast_power_kw` or something.

Reply:
go for
- `Forecasts.forecast_power_kw`
- `Generation.generation_power_kw`
?

Reply:
There's no right answer to this, it's the same as the argument about `table_name.table_uuid` vs `table_name.uuid` - we just have to pick one! Peter's suggestion above seems fine to me.